### PR TITLE
npm → pnpm, update dependencies

### PR DESCRIPTION
**Pull request by a-chabin**
- [x] npm → pnpm (`package.json` → `pnpm-lock.yaml`)
- [x] Update all dependencies (minor + patch, not major). Remove all `^` for consistency. 
- [x] Update README.md

Part of https://github.com/ekaterinburgdev/guides/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
 
-[Ссылка](https://vladatkishkin.gitbook.io/dokumentaciya-po-rukovodstvu/)
+**[guides.ekaterinburg.design](https://guides.ekaterinburg.design)**
 
-## Запуск
-
-Установка зависимостей
+## Development
 
+1. Add `.env.local` with back-end endpoints
```
Comment (AtkishkinVlad) [inline README.md:9]: А тут точно хочется добавить содержимое .env.local ?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 ```sh
-npm i
+NEXT_PUBLIC_HOST=https://guides-api-test.ekaterinburg.design
+NEXT_PUBLIC_IMAGE_DOMAIN=guides-api-test.ekaterinburg.design
 ```
 
-Разработка
+2. Install [Node.js](https://nodejs.org/en/download/) and [pnpm](https://www.npmjs.com/package/pnpm#user-content-install)
```
Comment (AtkishkinVlad) [inline README.md:15]: Можно добавить еще команды 

Windows – `choco install nodejs-lts`
Mac OS – `brew install node`
Linux – `sudo apt install nodejs`

Comment (ekbdev) [reply README.md:15]: I don't think we should be explaining how to install tools on different systems. This is not our responsibility.

Comment (a-chabin) [reply README.md:15]: I don't think we should be explaining how to install tools on different systems. This is not our responsibility.

File: README.md (hunk 1 above)
Comment (a-chabin) [reply README.md:9]: Yes, `.env.local` is in the `.gitignore` file and will not commit

Comment (a-chabin) [reply README.md:9]: Nextjs recommended using `.env.local`
https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

**Pull request merged**